src/all/hitomi/src/eu/kanade/tachiyomi/extension/all/hitomi/Hitomi.kt

```diff
@@ -93,12 +137,53 @@ class Hitomi(
         }
     }
 
-    private class SortFilter : Filter.Select<String>("Sort By", arrayOf("Updated", "Popularity"))
+    private class OrderFilter(val name: String, val order: OrderType) {
```

Comment:
since hitomi class is already too big, move the filter classes to a seperate file.